src/components/pages/home/hero/index.tsx

```diff
@@ -96,44 +96,33 @@ class HomeHero extends React.Component<HomeHeroProps, HomeHeroState> {
     this.handleLend = this.handleLend.bind(this);
   }
 
-  // getGanacheAddresses = async () => {
-  //   if (!this.ganacheProvider) {
-  //     this.ganacheProvider = getGanacheWeb3();
-  //   }
-  //   if (this.ganacheProvider) {
-  //     return await this.ganacheProvider.eth.getAccounts();
-  //   }
-  //   return [];
-  // };
-
   componentDidMount = async () => {
 
     // Get the contract instances for Ines (We'll just bake these in for now).
     const termsContractInstance = await getDeployedFromConfig('TermsContract', contractAddresses);
     const repaymentManagerInstance = await getDeployedFromConfig('RepaymentManager', contractAddresses);
 
     try {
-      const loanParams = await getLoanParams(termsContractInstance);
+      const loanPeriod = await getNumScheduledPayments(termsContractInstance);
+      const principalRequested = await getPrincipalRequested(termsContractInstance);
+      const interestRate = await getInterestRate(termsContractInstance);
+      const loanStartTimestamp = await getLoanStartTimestamp(termsContractInstance);
       const totaShares = await totalShares(repaymentManagerInstance);
-      const paymentToken = await getTokenDetailsFromAddress(loanParams.principalToken);
-
-      const principalRequested = loanParams.principalRequested;
-      // const payees = await repaymentManagerInstance.methods._payees();
-
+      const paymentToken = await getTokenDetailsFromAddress(await getPrincipalToken(termsContractInstance));
+      
       let loanEndTimestamp;
 
-      if (loanParams.loanStartTimestamp !== "0") {
+      if (loanStartTimestamp !== "0") {
```

Comment:
Is it more clear with LoanStatuses.NOT_STARTED than "0"?

Reply:
Nice catch, will update in next PR, so we can have this update available